Empty workspaces result in an error

```
Mar 15 18:02:50 kappa uwsm_sway-uwsm.desktop[469200]: Traceback (most recent call last):
Mar 15 18:02:50 kappa uwsm_sway-uwsm.desktop[469200]:   File "/nix/store/0kmq76vbnz4xkjly064axx0iqwca9vbs-sway-out/bin/.sway-out-wrapped", line 10, in <module>
Mar 15 18:02:50 kappa uwsm_sway-uwsm.desktop[469200]:     sys.exit(main())
Mar 15 18:02:50 kappa uwsm_sway-uwsm.desktop[469200]:              ~~~~^^
Mar 15 18:02:50 kappa uwsm_sway-uwsm.desktop[469200]:   File "/nix/store/982icsbh4ilbswa03p9m9pngi8pmgpxc-sway-out-env/lib/python3.14/site-packages/click/core.py", line 1485, in __call__
Mar 15 18:02:50 kappa uwsm_sway-uwsm.desktop[469200]:     return self.main(*args, **kwargs)
Mar 15 18:02:50 kappa uwsm_sway-uwsm.desktop[469200]:            ~~~~~~~~~^^^^^^^^^^^^^^^^^
Mar 15 18:02:50 kappa uwsm_sway-uwsm.desktop[469200]:   File "/nix/store/982icsbh4ilbswa03p9m9pngi8pmgpxc-sway-out-env/lib/python3.14/site-packages/click/core.py", line 1406, in main
Mar 15 18:02:50 kappa uwsm_sway-uwsm.desktop[469200]:     rv = self.invoke(ctx)
Mar 15 18:02:50 kappa uwsm_sway-uwsm.desktop[469200]:   File "/nix/store/982icsbh4ilbswa03p9m9pngi8pmgpxc-sway-out-env/lib/python3.14/site-packages/click/core.py", line 1873, in invoke
Mar 15 18:02:50 kappa uwsm_sway-uwsm.desktop[469200]:     return _process_result(sub_ctx.command.invoke(sub_ctx))
Mar 15 18:02:50 kappa uwsm_sway-uwsm.desktop[469200]:                            ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
Mar 15 18:02:50 kappa uwsm_sway-uwsm.desktop[469200]:   File "/nix/store/982icsbh4ilbswa03p9m9pngi8pmgpxc-sway-out-env/lib/python3.14/site-packages/click/core.py", line 1269, in invoke
Mar 15 18:02:50 kappa uwsm_sway-uwsm.desktop[469200]:     return ctx.invoke(self.callback, **ctx.params)
Mar 15 18:02:50 kappa uwsm_sway-uwsm.desktop[469200]:            ~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Mar 15 18:02:50 kappa uwsm_sway-uwsm.desktop[469200]:   File "/nix/store/982icsbh4ilbswa03p9m9pngi8pmgpxc-sway-out-env/lib/python3.14/site-packages/click/core.py", line 824, in invoke
Mar 15 18:02:50 kappa uwsm_sway-uwsm.desktop[469200]:     return callback(*args, **kwargs)
Mar 15 18:02:50 kappa uwsm_sway-uwsm.desktop[469200]:   File "/nix/store/982icsbh4ilbswa03p9m9pngi8pmgpxc-sway-out-env/lib/python3.14/site-packages/click/decorators.py", line 34, in new_func
Mar 15 18:02:50 kappa uwsm_sway-uwsm.desktop[469200]:     return f(get_current_context(), *args, **kwargs)
Mar 15 18:02:50 kappa uwsm_sway-uwsm.desktop[469200]:   File "/nix/store/982icsbh4ilbswa03p9m9pngi8pmgpxc-sway-out-env/lib/python3.14/site-packages/sway_out/main.py", line 96, in main_apply
Mar 15 18:02:50 kappa uwsm_sway-uwsm.desktop[469200]:     create_layout(connection, workspace_layout)
Mar 15 18:02:50 kappa uwsm_sway-uwsm.desktop[469200]:     ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Mar 15 18:02:50 kappa uwsm_sway-uwsm.desktop[469200]:   File "/nix/store/982icsbh4ilbswa03p9m9pngi8pmgpxc-sway-out-env/lib/python3.14/site-packages/sway_out/layout.py", line 271, in create_layout
Mar 15 18:02:50 kappa uwsm_sway-uwsm.desktop[469200]:     workspace_con.nodes
Mar 15 18:02:50 kappa uwsm_sway-uwsm.desktop[469200]: AssertionError: The workspace 2 (ID: 7) should not be empty at this point
```

--- a/src/sway_out/layout.py
+++ b/src/sway_out/layout.py
@@ -199,7 +199,8 @@ def create_layout(
         container_layout: ApplicationLaunchConfig | ContainerConfig,
     ) -> int:
         if isinstance(container_layout, ContainerConfig):
-            assert container_layout.children, "ContainerConfig must have children"
+            if not container_layout.children:
+                raise RuntimeError("Nested containers must have children")
             logger.debug(
                 f"Creating layout for container as {container_layout.layout} ..."
             )
@@ -253,6 +254,11 @@ def create_layout(
             logger.debug(f'Reached leaf "{result.name}" while creating layout')
             layouted_ids.add(result.id)
             return result.id
+
+    # If the layout is empty, we have nothing to do.
+    if not workspace_layout.children:
+        logger.info("The layout has no children, nothing to do here")
+        return
 
     # Check if the mark is already in use.
     marks_in_use = connection.get_marks()
@@ -384,7 +390,7 @@ def resize_layout(
             if not retry:
                 logger.debug(
                     f"Container {get_con_description(con)} resized successfully after "
-                    + f"{i+1}/{RESIZE_ATTEMPTS} attempts."
+                    + f"{i + 1}/{RESIZE_ATTEMPTS} attempts."
                 )
                 break
         else:

